contracts/nft.vy

```diff
@@ -1212,6 +1170,45 @@ def _check_owner():
     assert msg.sender == self.owner, "Ownable: caller is not the owner"
 
 
+@external
+def transfer_ownership_and_minter(new_owner: address):
```

Comment:
Note: this gives the owner power to arbitrarily mint any NFTs, if this isn't something we want then only the auction house contract can be minter

Reply:
Yeah, that possibility was okay to keep.